src/main/python/deep_qa/layers/overlap.py

```diff
@@ -0,0 +1,58 @@
+from keras import backend as K
+from keras.layers import Layer
+
+
+class Overlap(Layer):
+    """
+    This Layer takes 2 inputs: a tensor_a (e.g. a document) and a tensor_b
+    (e.g. a question). It returns a one-hot vector with the same shape as tensor_a,
+    indicating at each index whether the element in tensor_a appears in tensor_b.
+
+    Inputs:
+        - tensor_a: shape ``(batch_size, length_a)``
+        - tensor_b shape ``(batch_size, length_b)``
+
+    Output:
+        - One-hot indicating overlap: shape ``(batch_size, length_a, 2)``
```

Comment:
Why is the 2 there? Can't it just be `(batch_size, length_a)`? If it's going to be this way, can you just change the docs above to reflect that it is not the same shape as `tensor_a`.